plugins/woocommerce/src/Database/Migrations/CustomOrderTable/PostMetaToOrderMetaMigrator.php

```diff
@@ -40,7 +40,7 @@ public function __construct( $excluded_columns ) {
 	protected function get_meta_config(): array {
 		global $wpdb;
 		// TODO: Remove hardcoding.
-		$this->table_names = array(
+		$table_names = array(
```

Comment:
This looks fine for addressing the issue per file but we could eliminate duplication by removing these arrays and using the [datastore getters](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/src/Internal/DataStores/Orders/OrdersTableDataStore.php#L150)

Reply:
Thanks Ron. While working on this and the other PRs I noticed a few things (some flagged by PHPStan) that are worth fixing, and so I've started taking note of all of that. I'll ad this item to the list.

In the meantime, I'm going to merge this as is.

Reply:
Jorge has the brains. He uses PHPStan.